core/src/main/java/org/apache/accumulo/core/clientImpl/bulk/BulkImport.java

```diff
@@ -362,13 +362,15 @@ private static Map<String,Long> getFileLenMap(List<FileStatus> statuses) {
     return fileLens;
   }
 
-  private static Cache<String,Long> getPopulatedFileLenCache(Path dir, List<FileStatus> statuses) {
+  private static Cache<String,Long> getPopulatedFileLenCache(ClientContext ctx, Path dir,
+      List<FileStatus> statuses) {
     Map<String,Long> fileLens = getFileLenMap(statuses);
 
     Map<String,Long> absFileLens = new HashMap<>();
     fileLens.forEach((k, v) -> absFileLens.put(pathToCacheId(new Path(dir, k)), v));
 
-    Cache<String,Long> fileLenCache = Caffeine.newBuilder().build();
+    Cache<String,Long> fileLenCache =
```

Comment:
This is also a short lived cache, I think its created for each bulk v2 client request that inspects files.  If that is a problem could not emit metrics.

Reply:
Disabled metrics on this since you said it's short-lived, in 8bee224